### ISSUE TITLE
Error: Unable to resolve module `@env` from `src/utils.ts`: @env could not be found within the project.

### ISSUE THREAD

**Issue by JanithaR** [goatandsheep/react-native-dotenv#52]
I'm using yarn.

React Native info

> System:
>     OS: macOS 10.15.4
>     CPU: (8) x64 Intel(R) Core(TM) i5-8279U CPU @ 2.40GHz
>     Memory: 116.45 MB / 8.00 GB
>     Shell: 5.7.1 - /bin/zsh
>   Binaries:
>     Node: 12.16.3 - /usr/local/bin/node
>     Yarn: 1.22.4 - /usr/local/bin/yarn
>     npm: 6.14.4 - /usr/local/bin/npm
>     Watchman: 4.9.0 - /usr/local/bin/watchman
>   Managers:
>     CocoaPods: 1.9.3 - /usr/local/bin/pod
>   SDKs:
>     iOS SDK:
>       Platforms: iOS 13.5, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
>     Android SDK:
>       API Levels: 23, 27, 28, 29
>       Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.0
>       System Images: android-28 | Intel x86 Atom_64, android-28 | Google Play Intel x86 Atom_64
>       Android NDK: Not Found
>   IDEs:
>     Android Studio: 3.6 AI-192.7142.36.36.6392135
>     Xcode: 11.5/11E608c - /usr/bin/xcodebuild
>   Languages:
>     Java: 1.8.0_252 - /usr/bin/javac
>     Python: 2.7.16 - /usr/bin/python
>   npmPackages:
>     @react-native-community/cli: Not Found
>     react: 16.13.1 => 16.13.1 
>     react-native: 0.63.0 => 0.63.0 
>   npmGlobalPackages:
>     *react-native*: Not Found

My import statement 
`import { FIXER_KEY } from '@env';`
Typescript complains about this with

> Cannot find module '@env' or its corresponding type declarations

My .babelrc contents
`{
    "plugins": [
        [
            "module:react-native-dotenv",
            {
                "moduleName": "@env",
                "path": ".env",
                "blacklist": null,
                "whitelist": null,
                "safe": true,
                "allowUndefined": true
            }
        ]
    ]
}
`

I'm using version 

> "react-native-dotenv": "^2.3.0",

When I run the app I get

> Error: Unable to resolve module `@env` from `src/utils.ts`: @env could not be found within the project.

**Comment by mylonasg88** [comment]
Same here. I followed instructions:
```
 1. Clear watchman watches: watchman watch-del-all 
 2. Delete node_modules:
      rm -rf node_modules and run yarn install 
 3. Reset Metro's cache: yarn
      start --reset-cache 
 4. Remove the cache: rm -rf /tmp/metro-*
```

 I tried to install the `fs` module by `yarn add fs` but it kept failing with more errors.

**Comment by goatandsheep** [comment]
I believe this is a duplicate of #38

**Comment by gboyegadada** [comment]
@goatandsheep This is happening to me too. I don't think it's a duplicate issue. The other issue #38  seems to only affect production builds but I can't even get it working in dev / metro and Typescript complains about the import in vs code (same as with the OP).

I have also done everything @mylonasg88 mentioned. 

Here is my config file:

```
// babel.config.js ...

module.exports = {
  "presets": ["module:metro-react-native-babel-preset"],
  "plugins": [
    ["module:react-native-dotenv", {
      "moduleName": "@env",
      "path": ".env",
      "blacklist": null,
      "whitelist": null,
      "safe": true,
      "allowUndefined": true
    }]
  ]
};
```
![Image 12-08-2020 at 14 11](https://user-images.githubusercontent.com/6040727/90019153-ec46d100-dca5-11ea-8f1d-ec763a5e69ee.jpeg)

**Comment by ajaxjiang96** [comment]
I am using typescript and I've encountered the same issue after upgrading to `2.x`. Here is a workaround for it:
- Create a `types` folder in your project
- Inside that folder, create a `*.d.ts`file, say, `env.d.ts`
- in that file, declare a module as the following format:
```ts
declare module '@env' {
  export const API_BASE: string;
}
```
You should add all of your .env variables inside this module.

- Finally, add this folder into the `typeRoots` field in your `tsconfig.json` file:
```json
{
...
    "typeRoots": ["./src/types"],
...
}
```

**Comment by goatandsheep** [comment]
@LaureneFmy are you having a related issue? I noticed you 👎 the above solution.

**Comment by goatandsheep** [comment]
Anyone have a test environment https://snack.expo.io ?

**Comment by JanithaR** [comment]
> @JanithaR Anyone have a test environment https://snack.expo.io ?

If it's an example repo you are asking for, here it is. https://github.com/JanithaR/image-finder/blob/master/src/api.ts

**Comment by goatandsheep** [comment]
@JanithaR can you retry with 2.4.0?

**Comment by JanithaR** [comment]
@goatandsheep Seems to be the same. https://github.com/JanithaR/image-finder/blob/react-native-dotenv-update/package.json#L19

**Comment by goatandsheep** [comment]
Thank you for checking.

**Comment by goatandsheep** [comment]
@JanithaR one thing I noticed is that you have `safe: true`. I think maybe there's an issue with `safe` mode.

**Comment by alburdette619** [comment]
I go this error after enabling Hermes for the new version of react-native-reanimated.

**Comment by GavinThomas1192** [comment]
I am receiving the same error on a brand new native code react-native project. 
"react-native": "0.63.3",

**Comment by kalleriakronos24** [comment]
> @goatandsheep This is happening to me too. I don't think it's a duplicate issue. The other issue #38 seems to only affect production builds but I can't even get it working in dev / metro and Typescript complains about the import in vs code (same as with the OP).
> 
> I have also done everything @mylonasg88 mentioned.
> 
> Here is my config file:
> 
> ```
> // babel.config.js ...
> 
> module.exports = {
>   "presets": ["module:metro-react-native-babel-preset"],
>   "plugins": [
>     ["module:react-native-dotenv", {
>       "moduleName": "@env",
>       "path": ".env",
>       "blacklist": null,
>       "whitelist": null,
>       "safe": true,
>       "allowUndefined": true
>     }]
>   ]
> };
> ```
> 
> ![Image 12-08-2020 at 14 11](https://user-images.githubusercontent.com/6040727/90019153-ec46d100-dca5-11ea-8f1d-ec763a5e69ee.jpeg)

this fixed my issue, and i'm able to use this package properly now

**Comment by sabirahmed121** [comment]
I'm still facing this issue with version 2.4.2
Have also added the plugin in babel.config.js.

Anybody having any luck resolving this. 
Note I'm not using TS

**Comment by goatandsheep** [comment]
@sabirahmed121 try checking the #38 thread for non typescript issue

**Comment by sabirahmed121** [comment]
Was able to resolve this. So realised that my rebuild was not happening successfully even after the clearing commands.
The only way the rebuild was successful was when I made some edit to the code.

**Comment by mburnet6-ibm** [comment]
For others that encounter this issue, even in TS projects #38 can happen and we finally solved it with this solution: https://github.com/facebook/react-native/issues/29351#issuecomment-657929014

**Comment by chiawei0311** [comment]
> "plugins": [
>     ["module:react-native-dotenv", {
>       "moduleName": "@env",
>       "path": ".env",
>       "blacklist": null,
>       "whitelist": null,
>       "safe": true,
>       "allowUndefined": true
>     }]
>   ]

same to me, and need to build again.

**Comment by Kailash23** [comment]
Close all the windows and restart the laptop.

**Comment by kakaru1331** [comment]
in my case, @ajaxjiang96's solution works well.
thank you. :)

**Comment by PrincewillIroka** [comment]
After updating your babel.config.js file ensure to reset your cache with `yarn start --reset-cache` for changes to take place.

**Comment by f4z3k4s** [comment]
For me, the solution was to properly clear the cache. Note that I am using npm not yarn. In my case it took:

`npm cache clean --force`
`npm start -- --reset-cache` which in my setup (standard Expo setup) is equivalent to `react-native start "--reset-cache"`

**Comment by jilvanx** [comment]
> @goatandsheep This is happening to me too. I don't think it's a duplicate issue. The other issue #38 seems to only affect production builds but I can't even get it working in dev / metro and Typescript complains about the import in vs code (same as with the OP).
> 
> I have also done everything @mylonasg88 mentioned.
> 
> Here is my config file:
> 
> ```
> // babel.config.js ...
> 
> module.exports = {
>   "presets": ["module:metro-react-native-babel-preset"],
>   "plugins": [
>     ["module:react-native-dotenv", {
>       "moduleName": "@env",
>       "path": ".env",
>       "blacklist": null,
>       "whitelist": null,
>       "safe": true,
>       "allowUndefined": true
>     }]
>   ]
> };
> ```
> 
> ![Image 12-08-2020 at 14 11](https://user-images.githubusercontent.com/6040727/90019153-ec46d100-dca5-11ea-8f1d-ec763a5e69ee.jpeg)



> @goatandsheep This is happening to me too. I don't think it's a duplicate issue. The other issue #38 seems to only affect production builds but I can't even get it working in dev / metro and Typescript complains about the import in vs code (same as with the OP).
> 
> I have also done everything @mylonasg88 mentioned.
> 
> Here is my config file:
> 
> ```
> // babel.config.js ...
> 
> module.exports = {
>   "presets": ["module:metro-react-native-babel-preset"],
>   "plugins": [
>     ["module:react-native-dotenv", {
>       "moduleName": "@env",
>       "path": ".env",
>       "blacklist": null,
>       "whitelist": null,
>       "safe": true,
>       "allowUndefined": true
>     }]
>   ]
> };
> ```
> 
> ![Image 12-08-2020 at 14 11](https://user-images.githubusercontent.com/6040727/90019153-ec46d100-dca5-11ea-8f1d-ec763a5e69ee.jpeg)

I have this same issue and I'm not using TS. 
Can someone help me?

**Comment by jilvanx** [comment]
> @goatandsheep This is happening to me too. I don't think it's a duplicate issue. The other issue #38 seems to only affect production builds but I can't even get it working in dev / metro and Typescript complains about the import in vs code (same as with the OP).
> 
> I have also done everything @mylonasg88 mentioned.
> 
> Here is my config file:
> 
> ```
> // babel.config.js ...
> 
> module.exports = {
>   "presets": ["module:metro-react-native-babel-preset"],
>   "plugins": [
>     ["module:react-native-dotenv", {
>       "moduleName": "@env",
>       "path": ".env",
>       "blacklist": null,
>       "whitelist": null,
>       "safe": true,
>       "allowUndefined": true
>     }]
>   ]
> };
> ```
> 
> ![Image 12-08-2020 at 14 11](https://user-images.githubusercontent.com/6040727/90019153-ec46d100-dca5-11ea-8f1d-ec763a5e69ee.jpeg)

I have this same issue. I'm not using TS.
Can someone help me?

**Comment by goatandsheep** [comment]
* do you have a file named `.env`?
* Are you on the latest version?


If you answered yes to both of these, the easiest thing to do today is just change `moduleName` to `react-native-dotenv` and also in your file before we can continue to figure it out

**Comment by jilvanx** [comment]
> * do you have a file named `.env`?
> * Are you on the latest version?
> 
> If you answered yes to both of these, the easiest thing to do today is just change `moduleName` to `react-native-dotenv` and also in your file before we can continue to figure it out

I have .env file
version: "react-native-dotenv": "^2.5.3"
 
Do I change my babel file to look like this?

```
module.exports = {
  presets: ['module:metro-react-native-babel-preset'],
  plugins: [
    [
      'module:react-native-dotenv',
      {
        moduleName: 'react-native-dotenv',
      },
    ],
  ],
};
```

**Comment by goatandsheep** [comment]
Try it but then also change `@env` to `react-native-dotenv`. Basically you can rename your module whatever you want. I have a feeling this may reduce issues

**Comment by jilvanx** [comment]
> Try it but then also change `@env` in the document to `react-native-dotenv`. Basically you can rename your module whatever you want. I have a feeling this may reduce issues

My js file
`import { PUBLIC_KEY } from 'react-native-dotenv';`

My babel file
```
module.exports = {
  presets: ['module:metro-react-native-babel-preset'],
  plugins: [
    [
      'module:react-native-dotenv',
      {
        moduleName: 'react-native-dotenv',
      },
    ],
  ],
};
```
It's works for me :)

**Comment by f4z3k4s** [comment]
When above comments do not solve the error and you've reset every cache: try to reset the file cache of Chrome. I just happened to notice that the error only occured in debug mode, because the files that I was requiring the env variables in were cached. So this might help:

1. On your computer, open Chrome.
2. At the top right, click More More.
3. Click More tools and then Clear browsing data.
4. At the top, choose a time range. To delete everything, select All time.
5. Next to "Cookies and other site data" and "Cached images and files," check the boxes.
6. Click Clear data.

**Comment by GGrassiant** [comment]
Did what @jilvanx suggested and it worked perfectly. Thanks for the solution!
Since I am using typescript I just had to add an `env.d.ts` file with `declare module 'react-native-dotenv';`

**Comment by goatandsheep** [comment]
Awesome! There's also a definitely typed package for this repo that I haven't put anywhere in the docs https://www.npmjs.com/package/@types/react-native-dotenv

**Comment by GGrassiant** [comment]
Awesome! Thanks @goatandsheep

**Comment by BernardKintzing** [comment]
> Try it but then also change `@env` in the document to `react-native-dotenv`. Basically you can rename your module whatever you want. I have a feeling this may reduce issues

I have the same issue, when I rename from `@env` to `react-native-dotenv` the error is replaced with `Error: Unable to resolve module `fs` from `node_modules/react-native-dotenv/index.js`: fs could not be found within the project.` When I look for solutions to this error they say to replace `react-native-dotenv` with `@env` #20 #32

**Comment by BernardKintzing** [comment]
if your `babel.config.js` has the line `api.cache(true)` try rerunning and changing the line to `api.cache(false)`. If this works you can go back to `api.cache(true)` and change it when you update the file.

**Comment by mirandamon** [comment]
👋 Hi there - I'm having a similar issue as well in my typescript project. When I added the `env.d.ts` file, with these contents:
```
declare module '@env' {
  export const DETOX_ENV: string;
}
```
I no longer have any TypeScript errors. But when I try to build my app using `react-native bundle --reset-cache --entry-file='index.js' --bundle-output='./ios/TestApp/main.jsbundle' --dev false --platform ios`, I get this error:

```
error Unable to resolve module `@env` from `config.ts`: @env could not be found within the project.

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules: rm -rf node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*. Run CLI with --verbose flag for more details.
```

I've tried clearing watchman watches, node_modules, etc. but no luck. My `babel.config.js` file looks like this:

```module.exports = {
  presets: ['module:metro-react-native-babel-preset'],
  plugins: [
    'jest-hoist',
    [
      'module:react-native-dotenv',
      {
        moduleName: '@env',
        path: '.env',
        safe: false,
        allowUndefined: true,
      },
    ],
  ],
```


Basically, I have no TypeScript import errors now, but I do get a `Unable to resolve module `@env` from `config.ts`: @env could not be found within the project.` error when trying to build. Any ideas?

**Comment by goatandsheep** [comment]
@mirandamon where is your `.env` file relative to root?

**Comment by jffr** [comment]
For me it was a caching issue. I'm using NPM instead of Yarn. Thanks to @f4z3k4s for pointing me tothe right direction:

* Use `npm cache clean --force` to clean the cache folder for your packages
* Start the project with `expo start --clear`. This will also clean the Metro bundler cache

My config:

```javascript
presets: ['babel-preset-expo'],
plugins: [
  ['module:react-native-dotenv', {
    moduleName: '@env',
    path: '.env',
    blacklist: null,
    whitelist: null,
    safe: true,
    allowUndefined: true
  }]
]
```

**Comment by Shivam-Vishwasrao** [comment]
VS Code could be a little funky with how it imports modules from directories located above it, I had the same error and all I had to do to fix it was add a `./` before `src` directory. In this particular case it will look like `./src/utils.ts` and it should work. This may change on how many directories above is your `@env` file located.

**Comment by mrosendin** [comment]
Following the first recommendation in the readme, but this

```javascript
import { MODE } from "react-native-dotenv";
```

Gives this error:

`Module '"react-native-dotenv"' has no exported member 'MODE'.`

Also tried the second option in the readme, but then I get this error:

`Cannot find module '@env' or its corresponding type declarations`

**Comment by goatandsheep** [comment]
@mrosendin try changing line 2 to `api.cache(false);` and do you have a `.env` file with the value of `MODE` set?

**Comment by mrosendin** [comment]
@goatandsheep yes, I tried that too. I'm using `.env.development` and `.env.production`.

**Comment by goatandsheep** [comment]
@mrosendin try this https://github.com/goatandsheep/react-native-dotenv/wiki/Additional-Troubleshooting#2

Also if you're not already on latest, I recommend you use v3.1.1 as I made a very important priority fix

**Comment by mrosendin** [comment]
Just to clarify, my project is running fine, I'm just dealing with the VS Code errors below. I upgraded, and spent the past couple of hours trying various things. I am able to get the module to load successfully without the IDE error sometimes, but honestly I can't tell you why. Doesn't seem to matter because whenever I close and reopen the file it will display the error again.

<kbd>![Screen Shot 2021-07-16 at 9 12 37 PM](https://user-images.githubusercontent.com/3401412/126025132-5541ca84-a6ac-4e9e-9fd8-ad1f057ce428.png)</kbd>

<kbd>![Screen Shot 2021-07-16 at 9 13 18 PM](https://user-images.githubusercontent.com/3401412/126025167-d5df2487-0894-41b3-91c2-afbbaea4f920.png)</kbd>

I'm able to run the project, but the VS Code errors are a bit annoying. I'm silencing them with @ts-ignore for now.

**Comment by goatandsheep** [comment]
I'm sorry about all the difficulties 😢

**Comment by MrAdex77** [comment]
Hi, i was struggling 1 hours with this problem, nothing worked, finally some miracle happened when i changed in .env from 
`API_KEY=xx `
to 
`API_KEY = xxx`
and saved..

**Comment by og-abdul-mateen** [comment]
> I am using typescript and I've encountered the same issue after upgrading to `2.x`. Here is a workaround for it:
> 
> * Create a `types` folder in your project
> * Inside that folder, create a `*.d.ts`file, say, `env.d.ts`
> * in that file, declare a module as the following format:
> 
> ```ts
> declare module '@env' {
>   export const API_BASE: string;
> }
> ```
> 
> You should add all of your .env variables inside this module.
> 
> * Finally, add this folder into the `typeRoots` field in your `tsconfig.json` file:
> 
> ```json
> {
> ...
>     "typeRoots": ["./src/types"],
> ...
> }
> ```

I'm wondering how do you manage separate environments when using this approach? I mean how do you access the variables defined in `env.production` or `env.development` after declaring the `@env`?

**Comment by rush86999** [comment]
changing to ```"react-native-dotenv"``` works on ios but not on android. Any other suggestions?

**Comment by goatandsheep** [comment]
this ticket has been open for a while and that solution is in the README, so I'm closing this

**Comment by ivanherczeg** [comment]
> @f4z3k4s  When above comments do not solve the error and you've reset every cache: try to reset the file cache of Chrome. I just happened to notice that the error only occured in debug mode, because the files that I was requiring the env variables in were cached. So this might help:
> 
> 1. On your computer, open Chrome.
> 2. At the top right, click More More.
> 3. Click More tools and then Clear browsing data.
> 4. At the top, choose a time range. To delete everything, select All time.
> 5. Next to "Cookies and other site data" and "Cached images and files," check the boxes.
> 6. Click Clear data.

It is like wash your car, and take out the garbage.

**Comment by goatandsheep** [comment]
For now, yeah. A big babel update is coming soon that will fix this once and for all 🎈

**Comment by f4z3k4s** [comment]
> @ivanherczeg  It is like wash your car, and take out the garbage.

Yeah, pretty much. I know this is no place for ads, but I bought a new car since then, called [react-native-config](https://github.com/luggit/react-native-config). The extra it offers is that you can use the env vars in native as well. Plus, it just works.

 @goatandsheep no offense though, hope babel update will fix all ongoing issues. :)

**Comment by GermaVinsmoke** [comment]
I used this configuration and it worked for me.

**env.d.ts**
``` typescript
declare module 'react-native-dotenv' {
  export const API-KEY: string;
}
```

**babel.config.js**
``` javascript
module.exports = function (api) {
  api.cache(true);
  return {
    plugins: [
      [
        'module:react-native-dotenv',
        { moduleName: 'react-native-dotenv', path: '.env' },
      ],
    ],
  };
};
```

**index.ts**
``` typescript
import { API_KEY } from 'react-native-dotenv';
```

Not sure if it's good or not (security-wise) but it is working for me.

Only dependency which is needed is **react-native-dotenv**, not even the **@types/react-native-dotenv** is needed.

**Comment by samceena** [comment]
> I used this configuration and it worked for me.
> 
> **env.d.ts**
> 
> ```ts
> declare module 'react-native-dotenv' {
>   export const API-KEY: string;
> }
> ```
> 
> **babel.config.js**
> 
> ```js
> module.exports = function (api) {
>   api.cache(true);
>   return {
>     plugins: [
>       [
>         'module:react-native-dotenv',
>         { moduleName: 'react-native-dotenv', path: '.env' },
>       ],
>     ],
>   };
> };
> ```
> 
> **index.ts**
> 
> ```ts
> import { API_KEY } from 'react-native-dotenv';
> ```
> 
> Not sure if it's good or not (security-wise) but it is working for me.
> 
> Only dependency which is needed is **react-native-dotenv**, not even the **@types/react-native-dotenv** is needed.

thanks. this worked for me.

**Comment by ShivamJoker** [comment]
@goatandsheep have you purposefully kept the imports to `@env` in the readme? seems like it is causing lot of trouble to people.

Please update it with the package name.

**Comment by goatandsheep** [comment]
@ShivamJoker that would be a breaking change. I'd have to increment the major version number and already am working on a permanent caching fix

**Comment by ShivamJoker** [comment]
@goatandsheep I am not sure how cache is the problem I already did everything but still not sure how @env will work

**Comment by goatandsheep** [comment]
You're welcome to open a PR

**Comment by majorarif** [comment]
> For me, the solution was to properly clear the cache. Note that I am using npm not yarn. In my case it took:
> 
> `npm cache clean --force` `npm start -- --reset-cache` which in my setup (standard Expo setup) is equivalent to `react-native start "--reset-cache"`

Solved my issue.

**Comment by puckj** [comment]
> For me it was a caching issue. I'm using NPM instead of Yarn. Thanks to @f4z3k4s for pointing me to the right direction:
> 
> * Use `npm cache clean --force` to clean the cache folder for your packages
> * Start the project with `expo start --clear`. This will also clean the Metro bundler cache
> 
> My config:
> 
> ```js
> presets: ['babel-preset-expo'],
> plugins: [
>   ['module:react-native-dotenv', {
>     moduleName: '@env',
>     path: '.env',
>     blacklist: null,
>     whitelist: null,
>     safe: true,
>     allowUndefined: true
>   }]
> ]
> ```

work for me thank you very much xd

**Comment by LauColombini** [comment]
> Same here. I followed instructions:
> 
> ```
>  1. Clear watchman watches: watchman watch-del-all 
>  2. Delete node_modules:
>       rm -rf node_modules and run yarn install 
>  3. Reset Metro's cache: yarn
>       start --reset-cache 
>  4. Remove the cache: rm -rf /tmp/metro-*
> ```
> 
> I tried to install the `fs` module by `yarn add fs` but it kept failing with more errors.

Te amo chabon, sos crack, veni que te hago un bebe

**Comment by enisze** [comment]
The solutions are not working for me.

**Comment by LauColombini** [comment]
> The solutions are not working for me.

Share you code

**Comment by SeiwonPark** [comment]
> I used this configuration and it worked for me.
> 
> **env.d.ts**
> 
> ```ts
> declare module 'react-native-dotenv' {
>   export const API-KEY: string;
> }
> ```
> 
> **babel.config.js**
> 
> ```js
> module.exports = function (api) {
>   api.cache(true);
>   return {
>     plugins: [
>       [
>         'module:react-native-dotenv',
>         { moduleName: 'react-native-dotenv', path: '.env' },
>       ],
>     ],
>   };
> };
> ```
> 
> **index.ts**
> 
> ```ts
> import { API_KEY } from 'react-native-dotenv';
> ```
> 
> Not sure if it's good or not (security-wise) but it is working for me.
> 
> Only dependency which is needed is **react-native-dotenv**, not even the **@types/react-native-dotenv** is needed.

This worked for me too! thx
